flixel/FlxObject.hx

```diff
@@ -620,10 +626,11 @@ class FlxObject extends FlxBasic
 		last.x = x;
 		last.y = y;
 		
+		if (path != null && path.active)
+			path.update(elapsed);
```

Comment:
No, that would be a bit weird, since it got assigned to the `FlxObject` by the user. He might want to `restart()` it.